src/nnet3/nnet-training.cc

```diff
@@ -235,6 +235,45 @@ void ComputeObjectiveFunction(const GeneralMatrix &supervision,
         computer->AcceptOutputDeriv(output_name, &diff);
       break;
     }
+    case kLognormal: {
```

Comment:
SWBD best tdnn+chain :
/export/a10/dpovey/kaldi-tombstone/egs/swbd/s5c/exp/chain/tdnn_7b_sp

On Wed, Apr 13, 2016 at 2:25 PM, Hossein Hadian <notifications@github.com>
wrote:

> In src/nnet3/nnet-training.cc
> <https://github.com/kaldi-asr/kaldi/pull/414#discussion_r59599687>:
>
> > @@ -235,6 +235,45 @@ void ComputeObjectiveFunction(const GeneralMatrix &supervision,
> >          computer->AcceptOutputDeriv(output_name, &diff);
> >        break;
> >      }
> > +    case kLognormal: {
>
> One of the things that will affect this is whether or not it gives
> improvements on top of the chain models.
>
> Are there any chain trained models available so I can try ?
> @vijayaditya <https://github.com/vijayaditya> could you please point me
> to any chain trained models if available ?
> Thank you
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/414/files/ff4c55a18c594034f441df5ec0b8b7a8618b7be7#r59599687>
>